framework/source/class/qx/test/lang/Array.js

```diff
@@ -56,17 +56,17 @@ qx.Class.define("qx.test.lang.Array",
     
     
     testExclude : function() {
-      var a = [ 1, 2, 3, 4, 5 ];
-      qx.lang.Array.exclude(a, [ 2, 4 ]);
-      this.assertJsonEquals([ 1, 3, 5 ], a);
+      var a = [0,1,2,3,4,5,6,7,8,9];
+      qx.lang.Array.exclude(a, [1,2,3]);
+      this.assertJsonEquals([0,4,5,6,7,8,9], a);
       
-      var a = [ 1, 2, 3, 4, 5 ];
-      qx.lang.Array.exclude(a, new qx.data.Array([ 1, 3, 5 ]));
-      this.assertJsonEquals([ 2, 4 ], a);
+      var a = [0,1,2,3,4,5,6,7,8,9];
+      qx.lang.Array.exclude(a, new qx.data.Array([ 1,2,3 ]));
+      this.assertJsonEquals([0,4,5,6,7,8,9], a);
       
-      var a = new qx.data.Array([ 1, 2, 3, 4, 5 ]);
-      qx.lang.Array.exclude(a, [ 1, 3, 5 ]);
-      this.assertJsonEquals([ 2, 4 ], a.toArray());
+      var a = new qx.data.Array([0,1,2,3,4,5,6,7,8,9]);
+      qx.lang.Array.exclude(a, [1,2,3]);
+      this.assertJsonEquals([0,4,5,6,7,8,9], a.toArray());
```

Comment:
All of the former tests tested non-contiguous values being excluded (bad).
All of the new tests test only contiguous values being excluded (bad).
Shouldn't some of the tests test contiguous values being excluded, and others test non-contiguous values being excluded, to ensure that there's no adjacency or non-adjacency issues?